css/style.css

```diff
@@ -343,3 +343,64 @@ p {
   justify-content: space-around;
   gap: 2px;
 }
+
+.contact-description {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 40px;
+  text-align: center;
+  margin: 50px;
+  color: #42526e;
+  position: relative;
+  top: -30px;
+}
+
+.form {
+  padding: 0 20px;
+}
+
+.form-input {
+  width: 90%;
+  padding: 10px 10px;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin: 10px 20px;
+}
+
+.contact-button {
+  padding: 15px;
+  color: white;
+  border: none;
+  margin-top: 10px;
+  margin-left: 35%;
+  background-color: #36b37f;
+  border-radius: 4px;
+}
```

Comment:
- [ ] Kindly make sure to sticks to the design [templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) in Figma as much as possible, this includes the font, colours, border margin, padding.... your button doesn't really match the original design very much yet, please use the same value from the figma templete.
- eg: padding: 12px 12px 12px 12px;
- .......